skymp5-server/cpp/lib_espm/espm.cpp

```diff
@@ -4,8 +4,10 @@
 #include <cstring>
 #include <memory>
 #include <sparsepp/spp.h>
+#include <iostream>
 
 #include "espm.h"
+#include "Misc.h"
```

Comment:
Consider giving this file a more meaningful name like "GroupUtils" or something, but not just "Misc".
Also, you theoretically could join this file with `espm.h`, but it's your choice

Reply:
I could, but I son't like that this file is large. We should split stuff I think

Reply:
Yeah, `GroupUtils` sounds good, even though we have methods here, they're eventually end up being about groups